pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: instabug_dio_interceptor
 description:
   This package is an add on to instabug_flutter. It intercepts any requests performed
   with Dio Package and sends them to the report that will be sent to the dashboard.
-version: 2.1.1
+version: 2.2.0
```

Comment:
**question:** Should we go for a major version? Since we are supporting new major version of dio, it will be easier to grasp the v3 work with dio v5, v2 works with dio v4.

Reply:
You have a point here, but since this change introduces no breaking changes, I prefer a minor version bump in this case.